### ISSUE TITLE
Put github committer names in about box as contributors

### ISSUE THREAD

**Issue by activescott** [activescott/lessmsi#43]
pull them from github API and dispaly in about box. part of build script.

**Comment by activescott** [comment]
yeah love it. Maybe you can propose a plan first for how you'll implement it and we can go from there? Maybe just running git cli commands in CI to generate some contributors file that we can load in the about box or something? Hopefully something very simple, but 🤷‍♂️

**Comment by mega5800** [comment]
I'm pretty sure I can get the contributors GitHub link.

Regarding the all contributors,
I think this is a good idea, and will definitely motivate existing and future contributors.

In my opinion, the best way would be combining both approaches.

Personally, I'd feel honoured knowing that my name is shown in an application I've helped developing.

In the same spirit, I'd love to see a contributors section in the repo.

So, I'd like to start with completing this ticket and would suggest to create a new ticket for adding the all contributors section.

**Comment by activescott** [comment]
Sounds good to me.

**Comment by activescott** [comment]
Here are some relevant files we currently use for CI:
https://github.com/activescott/lessmsi/blob/master/appveyor.yml: This one is basically the confirmation for appveyer telling it what to run. Essentially it runs some scripts we tell it to.
https://github.com/activescott/lessmsi/blob/master/src/.build/lessmsi.msbuild is the bulk of the logic for building the app and creating artifacts. You can run this script the same way that appveyor does locally, but you won’t have the keys to update github or chocolatey.
For adding something to the about box, my guess is you’ll have to do something in the msbuild script or maybe better to run something before it runs as a separate script to create or update an additional artifact (I think the aboutbox is really just showing some rtf file or something like that).

However, to update the readme, I’d just use GitHub actions or add some bot to the repo with config.
